task03recorrected/src/pages/Login.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react";
+import { Col, Container, Row, Image } from "react-bootstrap";
+import { useNavigate } from "react-router-dom";
+import photo from "../assets/login.png";
+import axios from "axios";
+
+const Login = () =>
+{
+    const navigate = useNavigate();
+    const [user, setUser] = useState({
+        email: "",
+        password: ""
+    })
+    const url = 'http://restapi.adequateshop.com/api/authaccount/login';
+    const handleChange = (e: { target: { name: any; value: any; }; }) =>
+    {
+        const {name, value} = e.target;
+        setUser({
+            ...user,
+            [name]: value
+        })
+    }
+
+    const login = async () => {
+        try{
+            const response = await axios.post(`${url}`, user );
+            const msg = response.data.message;
+            if (msg === 'success')
+            {
+                navigate('/home', { state: { name: user.email, password: user.password }});
```

Comment:
don't need password to homepage. and don't try to pass data via navigation while u have easy methods to pass data.